symyonf console make:entity returns error after asking to "mark this class as an API Platform resource"

**Symfony version(s) affected**: 5.3.0-BETA2

**Description**

```shell
michaelbrauner@Michaels-MacBook-Pro cardedu % symfony console make:entity               

 Class name of the entity to create or update (e.g. GentleChef):
 > Source

 Mark this class as an API Platform resource (expose a CRUD API for it) (yes/no) [no]:
 > 


In Generator.php line 140:
                                          
  Warning: Uninitialized string offset 0                          

```

```shell
michaelbrauner@Michaels-MacBook-Pro cardedu % symfony console make:entity

 Class name of the entity to create or update (e.g. VictoriousGnome):
 > Source

 Mark this class as an API Platform resource (expose a CRUD API for it) (yes/no) [no]:
 > yes


                                                                                                                        
 [ERROR] "App\Entity\1" is not valid as a PHP class name (it must start with a letter or underscore, followed by any    
         number of letters, numbers, or underscores)                                                                    
                                                                                                                        

exit status 1

```

**Additional context**

--- a/src/Maker/MakeEntity.php
+++ b/src/Maker/MakeEntity.php
@@ -114,32 +114,32 @@ final class MakeEntity extends AbstractMaker implements InputAwareMakerInterface
 
         $argument = $command->getDefinition()->getArgument('name');
         $question = $this->createEntityClassQuestion($argument->getDescription());
-        $value = $io->askQuestion($question);
+        $entityClassName = $io->askQuestion($question);
 
-        $input->setArgument('name', $value);
+        $input->setArgument('name', $entityClassName);
 
         if (
             !$input->getOption('api-resource') &&
             class_exists(ApiResource::class) &&
-            !class_exists($this->generator->createClassNameDetails($value, 'Entity\\')->getFullName())
+            !class_exists($this->generator->createClassNameDetails($entityClassName, 'Entity\\')->getFullName())
         ) {
             $description = $command->getDefinition()->getOption('api-resource')->getDescription();
             $question = new ConfirmationQuestion($description, false);
-            $value = $io->askQuestion($question);
+            $isApiResource = $io->askQuestion($question);
 
-            $input->setOption('api-resource', $value);
+            $input->setOption('api-resource', $isApiResource);
         }
 
         if (
             !$input->getOption('broadcast') &&
             class_exists(Broadcast::class) &&
-            !class_exists($this->generator->createClassNameDetails($value, 'Entity\\')->getFullName())
+            !class_exists($this->generator->createClassNameDetails($entityClassName, 'Entity\\')->getFullName())
         ) {
             $description = $command->getDefinition()->getOption('broadcast')->getDescription();
             $question = new ConfirmationQuestion($description, false);
-            $value = $io->askQuestion($question);
+            $isBroadcast = $io->askQuestion($question);
 
-            $input->setOption('broadcast', $value);
+            $input->setOption('broadcast', $isBroadcast);
         }
     }
 

--- a/tests/Maker/MakeEntityTest.php
+++ b/tests/Maker/MakeEntityTest.php
@@ -593,5 +593,28 @@ class MakeEntityTest extends MakerTestCase
                 $this->assertStringContainsString(\PHP_VERSION_ID >= 80000 ? '#[Broadcast]' : '@Broadcast', $content);
             }),
         ];
+
+        yield 'entity_new_with_api_and_broadcast_dependencies' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeEntity::class),
+            [
+                // entity class name
+                'User',
+                // Mark the entity as not an API Platform resource
+                'n',
+                // Mark the entity as not broadcasted
+                'n',
+                // add not additional fields
+                '',
+            ])
+            ->setRequiredPhpVersion(70200)
+            ->addExtraDependencies('api')
+            ->addExtraDependencies('ux-turbo-mercure')
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeEntity')
+            ->configureDatabase()
+            ->updateSchemaAfterCommand()
+            ->assert(function (string $output, string $directory) {
+                $this->assertFileExists($directory.'/src/Entity/User.php');
+            }),
+        ];
     }
 }

